### PR TITLE
Compartment docs

### PR DESCRIPTION
**Pull request by LeilyR**
* [ ] Flake8 passes (`flake8 . --exclude=.venv,.build,planemo_test_env,build --ignore=E501,F403,E402,F999,F405,E712`)
* [ ] Local tests pass (`py.test hicexplorer --doctest-modules`)

### CODE REVIEW

File: docs/content/tools/hicCompartmentsPolarization.rst [hunk 1]
```diff
@@ -6,3 +6,19 @@ hicCompartmentsPolarization
...
+   To our knoweldege this method has been first introduced by Wibke Schwarzer
+   et al. 2017 (Nature. 2017 Nov 2; 551(7678): 51–56). In this method, a
+   global (genome-wide) strength for compartmentalization is computed as
+   (AA + BB) / (AB + BA)
+   after rearranging the bins of obs/exp based on their corresponding pc1 values.
+   For this purpose, first pc1 values are reordered incrementally, then the same
+   order of bins is used to rearrange the bins in obs/exp matrix.
+   .. code:: bash
+
+   $ hicGlobalInteraction --obsexp_matrices obsExpMatrix.h5 --pca pc1.bedgraph
```
Comment (joachimwolff) [inline docs/content/tools/hicCompartmentsPolarization.rst]: What is hicGlobalInteraction?

File: hicexplorer/hicCompartmentsPolarization.py [hunk 2]
```diff
@@ -13,7 +13,9 @@ def parse_arguments():
     parser = argparse.ArgumentParser(
         description="""
         Rearrange the average interaction frequencies using the first PC values
-        to represent the global compartmentalization signal
+        to represent the global compartmentalization signal. To our knowledge
+        this has been first introduced and implemented by Wibke Schwarzer et
+        al. 2017 (Nature. 2017 Nov 2; 551(7678): 51–56)
 
         $ hicGlobalInteraction --obsexp_matrices obsExpMatrix.h5 --pca pc1.bedgraph\
```
Comment (joachimwolff) [inline hicexplorer/hicCompartmentsPolarization.py]: hicGlobalInteraction?

File: docs/content/tools/hicCompartmentsPolarization.rst [hunk 3]
```diff
@@ -6,3 +6,19 @@ hicCompartmentsPolarization
 .. argparse::
    :ref: hicexplorer.hicCompartmentsPolarization.parse_arguments
    :prog: hicCompartmentsPolarization
+
+
+   Applying PCA to compute the global compartmentalization signal
+   -----------------------------
+
+   To our knoweldege this method has been first introduced by Wibke Schwarzer
```
Comment (joachimwolff) [inline docs/content/tools/hicCompartmentsPolarization.rst]: typo

File: docs/content/tools/hicCompartmentsPolarization.rst (hunk 1 above)
Comment (LeilyR) [reply docs/content/tools/hicCompartmentsPolarization.rst]: typo, fixed it

File: hicexplorer/hicCompartmentsPolarization.py (hunk 2 above)
Comment (LeilyR) [reply hicexplorer/hicCompartmentsPolarization.py]: fixed

File: docs/content/tools/hicCompartmentsPolarization.rst [hunk 4]
```diff
@@ -6,3 +6,19 @@ hicCompartmentsPolarization
...
+   To our knowledge this method has been first introduced by Wibke Schwarzer
+   et al. 2017 (Nature. 2017 Nov 2; 551(7678): 51–56). In this method, a
+   global (genome-wide) strength for compartmentalization is computed as
+   (AA + BB) / (AB + BA)
+   after rearranging the bins of obs/exp based on their corresponding pc1 values.
+   For this purpose, first pc1 values are reordered incrementally, then the same
+   order of bins is used to rearrange the bins in obs/exp matrix.
+   .. code:: bash
+
+   $ hicCompartmentsPolarization --obsexp_matrices obsExpMatrix.h5 --pca pc1.bedgraph
```
Comment (joachimwolff) [inline docs/content/tools/hicCompartmentsPolarization.rst]: I think this will not be rendered as it should be. Please check compile it locally to verify.

**Pull request merged**